.github/actions/k-bench/action.yml

```diff
@@ -0,0 +1,131 @@
+name: k-bench
+description: "Run K-Bench"
+
+inputs:
+  cloudProvider:
+    description: "Which cloud provider to use."
+    required: true
+  kubeconfig:
+    description: "The kubeconfig of the cluster to test."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup python
+      uses: actions/setup-python@b55428b1882923874294fa556849718a1d7f2ca5 # tag=v4.2.0
+      with:
+        python-version: "3.10"
```

Comment:
Personally I would move the Python setup to the workflow itself just to have most setup steps at one place (and given that having Python can be useful for other things). 

But I don't have a hard preference if this is the only action really using it so far.

Theoretically you could likely even skip it since Python is likely preinstalled in the runner images, but there might be reasons we want to pin this for reproducibility of the results (in case Python could affect that).